BigBaseV2/src/api/remote.hpp

```diff
@@ -49,6 +49,8 @@ namespace big::remote
 			std::string local_etag = "";
 			std::string remote_etag = "";
 
+			std::vector<std::string> headers = { "User-Agent: Mozilla/5.0 (Windows; U; Windows NT 6.1; en-US) AppleWebKit/533.2 (KHTML, like Gecko) Chrome/6.0" };
```

Comment:
Use `constexpr std::vector<std::string>`  the value is never modified and the compiler can optimize this to not have to construct the vector each time.